controllers/release/adapter.go

```diff
@@ -761,20 +764,27 @@ func (a *adapter) createManagedCollectorsPipelineRun(releasePlanAdmission *v1alp
 
 // createTenantCollectorsPipelineRun creates a PipelineRun to run the collectors Pipeline for collectors in the ReleasePlan.
 func (a *adapter) createTenantCollectorsPipelineRun(releasePlan *v1alpha1.ReleasePlan, releasePlanAdmission *v1alpha1.ReleasePlanAdmission) (*tektonv1.PipelineRun, error) {
+	var url string
 	var revision string
 	var err error
+	var err2 error
 
 	if releasePlanAdmission != nil {
-		revision, err = releasePlanAdmission.Spec.Pipeline.PipelineRef.GetRevision()
+		url, err = releasePlanAdmission.Spec.Pipeline.PipelineRef.GetUrl()
+		revision, err2 = releasePlanAdmission.Spec.Pipeline.PipelineRef.GetRevision()
```

Comment:
I would add the defaults before the `if releasePlanAdmission != nil`  then check call the `...GetUrl`  and `...GetRevision` using the same error.

Something like this
```
defaulRepo = "git://..." // better getting from config btw
defaultRevision = "production"

if rpa != nil {
  url, err = ...GetUrl()
  if err != nil {
    url = defaultUrl
  }
  resivion, err = ...GetRevision()
  if err != nil {
    revision = defaultRevision
  }
}
``

Reply:
> The TenantPipeline should use the values provided in the ReleasePlan, not in RPA

There is no way to specify the repo/revision to use for a Tenant _collector_. 

I think what you are saying is try to take the repo/rev from the Tenant pipeline **pipelineref** (if one if defined) and then fallback on the Managed pipeline  **pipelineref** value in the RPA?

Reply:
Conceptually, yes. Because the idea of the Tenant Pipeline is the user leveraging his own name/workspace to configure/execute his Pipeline, so adding the RPA dependency here seems wrong to me.

Also you don´t need a fallback, because if it is a Tenant Pipeline, the url and revision should be set, otherwise the Pipeline won´t even execute.

Reply:
can we tackle this in a future PR? right now, we need this to help speed up the development of the collector feature.